src/main/scala/nl/knaw/dans/easy/agreement/pdfgen/Placeholders.scala

```diff
@@ -0,0 +1,134 @@
+/**
+ * Copyright (C) 2019 DANS - Data Archiving and Networked Services (info@dans.knaw.nl)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package nl.knaw.dans.easy.agreement.pdfgen
+
+import java.io.{ File => JFile }
+import java.net.URLEncoder
```

Comment:
`URLEncoder` will encode `10.17026/dans-xn3-ptsa` into `10.17026%2Fdans-xn3-ptsa`. Using `dans-scala-lib`'s encoding feature (using Guava) will however result in `10%2E17026%2Fdans%2Dxn3%2Dptsa` and a url like https://doi.org/10%2E17026%2Fdans%2Dxn3%2Dptsa in the PDF, rather than https://doi.org/10.17026%2Fdans-xn3-ptsa. I guess either one is fine, and even the not-encoded version (https://doi.org/10.17026/dans-xn3-ptsa) seems OK for doi.org.

Perhaps I like the last one best. Maybe we should remove the encoding entirely here...

Reply:
Agreed, we don't really use any characters that require encoding.